packages/cosmjs/src/core/queryClient/queryClient.ts

```diff
@@ -0,0 +1,399 @@
+import { seiprotocol } from '@sei-js/proto';
```

Comment:
do we have to manually import types like this? aren't the query clients already typed from @sei-js/proto?